pkg/bundle/sync_test.go

```diff
@@ -362,6 +463,25 @@ func Test_syncTarget(t *testing.T) {
 				} else {
 					assert.NotContains(t, configMap.OwnerReferences, expectedOwnerReference)
 				}
+
+				jksData, jksExists := configMap.BinaryData[jksKey]
+				if test.expJks {
+					assert.True(t, jksExists, "jks should be present in configmap")
+					reader := bytes.NewReader(jksData)
+					ks := jks.New()
+					err := ks.Load(reader, []byte(defaultJksPassword))
+					assert.Nil(t, err)
+					entryNames := ks.Aliases()
+					assert.Len(t, entryNames, 1)
+					assert.True(t, ks.IsTrustedCertificateEntry(entryNames[0]))
+					// Safe to ignore errors here, we've tested that it's present and a TrustedCertificateEntry
+					cert, _ := ks.GetTrustedCertificateEntry(entryNames[0])
+					// Only one certificate block for this test, so we can safely ignore the `remaining` byte array
+					p, _ := pem.Decode([]byte(data))
+					assert.Equal(t, p.Bytes, cert.Certificate.Content)
+				} else {
+					assert.False(t, jksExists, "jks should not be present in configmap")
+				}
```

Comment:
suggestion: Would it be cleaner for us to `assert.Equal` on `jksExists` rather than having the else block? I think that would more closely match the usual style I expect from Golang

```suggestion

				jksData, jksExists := configMap.BinaryData[jksKey]
				assert.Equal(t, test.expJKS, jksExists)

				if test.expJks {
					reader := bytes.NewReader(jksData)
					ks := jks.New()
					err := ks.Load(reader, []byte(defaultJksPassword))
					assert.Nil(t, err)
					entryNames := ks.Aliases()
					assert.Len(t, entryNames, 1)
					assert.True(t, ks.IsTrustedCertificateEntry(entryNames[0]))
					// Safe to ignore errors here, we've tested that it's present and a TrustedCertificateEntry
					cert, _ := ks.GetTrustedCertificateEntry(entryNames[0])
					// Only one certificate block for this test, so we can safely ignore the `remaining` byte array
					p, _ := pem.Decode([]byte(data))
					assert.Equal(t, p.Bytes, cert.Certificate.Content)
				}
```

Reply:
Yes, that's much cleaner - thanks!